### DISCUSSION TITLE
How can I externalize a certain package and require them at runtime?

### DISCUSSION THREAD

**Discussion by neko-para** [yao-pkg/pkg#74] [Q&A] [answered]
I have a native addon that also dynamic linking to some dlls, and pkg just copied .node itself which doesn't work. I'd like to ask pkg to ignore this package and I'll copy node_modules beside.

**Comment by robertsLando** [comment, marked as answer]
Starting from 5.12.0 you can use `--ignore` flag to ignore certain files from pkg build, see https://github.com/yao-pkg/pkg/issues/68
